app/error_handlers.py

```diff
@@ -0,0 +1,9 @@
+from flask import current_app as app
+from flask import render_template, request
+
+
+# authentication error
+@app.errorhandler(403)
```

Comment:
You might consider using Flask status codes to make the error codes more readable.

https://flask-api.github.io/flask-api/api-guide/status-codes/